Scroller/Scroller.js

```diff
@@ -53,7 +53,33 @@ let scrollerId = 0;
  * @ui
  * @public
  */
-let Scroller = ({'aria-label': ariaLabel, hoverToScroll, ...rest}) => {
+const scrollerDefaultProps = {
```

Comment:
How about moving this definition before the jsdoc?

Reply:
It's moved.